### PR TITLE
Various tiny mofidications

### PR DESCRIPTION
**Pull request by Ardakilic**
* License added
* Multi Language support
* Code cleaned up and comments translated
* More configuration parameters are put so index.js doesn't need modification for e.g: limit changes.
* To check a lower amount, instead of multiplying by 2, now it sums it with a minimum_amount new variable. If the delimiter is bigger, readjusting may put order too low or high.

Before merge, I'd appreciate if you could test this so I didn't mess up anything.

### CODE REVIEW

**Comment by demirhancosku** [comment]
@Ardakilic Thank you for your contribution, I'll check your pr as soon as possible.

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const Kucoin = require('./src/api'),
     config = require('./config');
 
-const lang = require('./src/lang/' + config.lang);
+const lang = require('./src/lang/' + config.ui.lang);
```
Comment (demirhancosku) [inline index.js]: You can merge the const variable definition of lang with the others :) It'll work anyway.

Comment (Ardakilic) [reply index.js]: Done in https://github.com/demirhancosku/kucoin-api-nodejs/pull/1/commits/8ef3115e3ee111d64b7198fa41a0da2fdcc67223 :)

**Comment by demirhancosku** [comment]
@Ardakilic You've done a great job, thank you for your contribution again!

**Pull request merged**